### PR TITLE
Hotfix/test update

### PR DESCRIPTION
**Pull request by bfloreaDVSA**
added defect ids for ignored tests
minor changes

### CODE REVIEW

**Review by jgimber-deloitte — APPROVED** [review]

**Review by dpanzaru-deloitte — DISMISSED** [review]

**Comment by dpanzaru-deloitte** [comment]
Before merging, the conflict need to be resolved.

File: src/main/java/clients/TestTypesClient.java [hunk 1]
```diff
@@ -89,6 +89,7 @@ private Response callTestTypes(String id, TestTypeQueryParam testTypeQueryParam)
     private Response callGetTestTypes(Filter filter) {
         Response response = given().filters(filter)
                 .contentType(ContentType.JSON)
+                .log().all()
```
Comment (jgimber-deloitte) [inline src/main/java/clients/TestTypesClient.java]: This won't put bearer tokens into the debug will it?  That shouldn't be in there.

File: src/test/java/defects/GetDefectsNoData.java [hunk 2]
```diff
@@ -26,7 +26,7 @@
     @Steps
     DefectsSteps defectsSteps;
 
-    @Ignore
+    @Ignore ("no data test")
```
Comment (jgimber-deloitte) [inline src/test/java/defects/GetDefectsNoData.java]: Is there a bug ref for this being disabled?

File: src/test/java/testresults/PostTestResultsNegMainLvlSubmitted.java [hunk 3]
```diff
@@ -505,9 +504,7 @@ public void testResultsMissingTestStatus() {
         testResultsSteps.validatePostErrorData("testStatus", "should be one of [\"submitted\", \"cancelled\"]");
     }
 
-
-    //TODO - possible problem with gateway
-    @Ignore ("null testStatus should have returned an error - instead returns 502")
+    @Ignore ("null testStatus should have returned an error - instead returns 502 - defect Id CVSB-9015")
```
Comment (jgimber-deloitte) [inline src/test/java/testresults/PostTestResultsNegMainLvlSubmitted.java]: Remember to add a subtask to 9015 to re-enable the tests once fixed ;)

File: src/test/java/testresults/PostTestResultsPozMainLvlSubmitted.java [hunk 4]
```diff
@@ -178,7 +178,7 @@ public void testResultsRandomTesterStaffId() {
         validateSavedData();
     }
 
-    @Ignore
+    @Ignore ("empty testerStaffId should be accepted but returned an error - instead returns bad Gateway - defect id CVSB-9018")
```
Comment (jgimber-deloitte) [inline src/test/java/testresults/PostTestResultsPozMainLvlSubmitted.java]: Remember to add a subtask to this issue, to re-enable the tests...

**Review by jgimber-deloitte — CHANGES_REQUESTED** [review]
Comments included.  Nothing major.

File: src/main/java/clients/TestTypesClient.java (hunk 1 above)
Comment (bfloreaDVSA) [reply src/main/java/clients/TestTypesClient.java]: fixed

File: src/test/java/defects/GetDefectsNoData.java (hunk 2 above)
Comment (bfloreaDVSA) [reply src/test/java/defects/GetDefectsNoData.java]: no, it was disabled long time ago, it does not run on local, maybe we should at some point enable it and see how it goes on CI

**Review by jgimber-deloitte — APPROVED** [review]

**Review by dpanzaru-deloitte — APPROVED** [review]

**Pull request merged**